### DISCUSSION TITLE
How do you allow crawlers to bypass the Attack Challenge Mode?

### DISCUSSION THREAD

**Discussion by Sir-Baconn** [vercel/community#7179] [Feature Question] [answered]
### Question

I have a nextjs application with Open Graph meta tags configured so that social media crawlers can display an image in their card/preview. This works as expected when Attack Challenge Mode is off.

When I turn Attack Challenge Mode on, social media platforms are not reading any of the meta tags as no image or text is displayed in the card/preview. I assume this is because the crawler cannot pass the verification check. Thus, I have added this rule to the firewall rules:
![image](https://github.com/vercel/community/assets/11286486/21ed748f-f542-498b-8be8-cb4c36aac9d1)

Which to my understanding would allow, in this case, discord to bypass the check but it seems like it does not.

Referencing https://udger.com/resources/ua-list/crawlers for the list of user-agent strings.

Am I misunderstanding how this all works?

Thanks!

**Comment by leerob** [comment, marked as answer]
Attack Challenge Mode is not meant to be enabled all of the time, only when you are under attack. Instead, it sounds like you want a more granular rule to challenge traffic. You can have one custom rule, that runs first, which bypasses the Discord user agents. Then, you can have a rule after which includes a challenge.

**Reply by Sir-Baconn** [reply]
Thank you!

[a comment by an automated account is not shown]
